wild_lib/src/lib.rs

```diff
@@ -106,6 +110,9 @@ fn link<S: storage::StorageModel, A: arch::Arch>(args: &Args) -> crate::error::R
     let output_file = output.write::<S, A>(&layout)?;
     diff::maybe_diff()?;
 
+    if let Some(done_callback) = done_closure {
```

Comment:
If this is not the right place to signal parent, might need some guidance to find it.

Reply:
I think after the line `shutdown::free_output(output_file);` below would be safest